swan-cern/files/swan_config_cern.py

```diff
@@ -24,6 +24,16 @@ def get_swan_user_pod(self):
         # init user containers (notebook and side-container)
         self._init_eos_containers(eos_secret_name)
 
+        if self._gpu_enabled():
```

Comment:
I still believe we need to rethink the gpu logic. If it belongs here or in the spawner, we need to see.
But, this might need to be optional as well.

Reply:
ok, but i suggest we merge this and revisit later, what do you think?

Reply:
yes